addon/components/single-event-objective-list.js

```diff
@@ -3,6 +3,7 @@ import { tracked } from '@glimmer/tracking';
 
 export default class SingleEventObjectiveList extends Component {
   @tracked groupByCompetencies = true;
+  @tracked isExpanded = false;
```

Comment:
I *think* this would work:
```suggestion
  @tracked isExpanded = this.args.isExpanded ?? false;
```

Reply:
:heavy_check_mark: went with something like that.